provision.yaml

```diff
@@ -0,0 +1,16 @@
+---
+default:
+  provisioner: docker 
+  images: ['waffleimage/debian8']
+travis_deb:
+  provisioner: docker
+  images: ['debian:8', 'debian:9', 'ubuntu:14.04', 'ubuntu:16.04', 'ubuntu:18.04']
+travis_el:
+  provisioner: docker_exp
+  images: ['centos:6', 'centos:7', 'oraclelinux:6', 'oraclelinux:7', 'scientificlinux/sl:6', 'scientificlinux/sl:7']
+release_checks:
+  provisioner: vmpooler
+  images: ['redhat-6-x86_64', 'redhat-7-x86_64', 'redhat-8-x86_64', 'centos-6-x86_64', 'centos-7-x86_64', 'oracle-6-x86_64', 'oracle-7-x86_64', 'scientific-6-x86_64', 'scientific-7-x86_64', 'debian-8-x86_64', 'debian-9-x86_64', 'debian-10-x86_64',  'ubuntu-1404-x86_64', 'ubuntu-1604-x86_64', 'ubuntu-1804-x86_64', 'sles-11-x86_64', 'sles-12-x86_64', 'sles-15-x86_64', 'win-2008r2-x86_64', 'win-2012r2-x86_64', 'win-2016-x86_64', 'win-10-pro-x86_64']
```

Comment:
hey @michaeltlombardi ,

Comparing with metadata, redhat5, centos5, oracle5 are missing. Also sles, for windows some versions and solaris. Do we need to add those platforms also for testing or to update metadata 🤔 ?

Cheers

Reply:
would you also add a screenshot with the successful test report?

Reply:
PIpelines is failing
https://pipelines.puppet.com/team-modules/builds/603037